Assets/Scripts/Ingredients/InteractableBehaviour.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
+using UnityEngine;
+
+public class InteractableBehaviour : MonoBehaviour
+{
+    [SerializeField] GameObject hoverObject;
+    Transform worldCanvas => GameObject.Find("WorldCanvas").transform;
+    [SerializeField] Transform player;
+    GameObject hoverObjectInstance;
+
+    // Start is called before the first frame update
+    void Start()
```

Comment:
```suggestion
    private void Start()
```
Bij al je functies kan je niet zien of het public of private is, dit zou je kunnen aanpassen, want het is wel netter